src/components/WorkoutInputDialog.tsx

```diff
@@ -13,12 +13,12 @@ import {
   KeyboardDatePicker,
   MuiPickersUtilsProvider
 } from '@material-ui/pickers';
-
 import { WorkoutTypes } from '../domain/workout';
 import { addWorkout } from '../services/workoutService';
 
 interface WorkoutInputDialogProps {
-  isOpen: boolean;
+  isDialogOpen: boolean;
```

Comment:
FYI the prop name does not need to match the variable itself that is passed in as the prop. 👀